src/main/java/com/powsybl/openloadflow/network/ShuntVoltageControl.java

```diff
@@ -19,7 +19,7 @@
     private final List<LfShunt> controllers = new ArrayList<>();
 
     public ShuntVoltageControl(LfBus controlled, double targetValue) {
-        super(controlled, targetValue);
+        super(controlled, targetValue, 0.0);
```

Comment:
Here we set by  the value of dead band as zero(is a fixed value, should ideally be used as a parameter or as a property that could be set by the user), is that correct? Furthermore, in TransformerVoltagecontrolouterloop , this value is used to check if voltage lies beyond this point, in order to set the voltageControl as true